feature-list-streams/src/main/java/com/codandotv/streamplayerapp/feature_list_streams/profile/presentation/screens/ProfilePickerStreamScreen.kt

```diff
@@ -0,0 +1,279 @@
+                )
+
+                BoxWithConstraints(
+                    Modifier
+                        .fillMaxSize()
+                        .padding(top = 100.dp)
+                        .graphicsLayer(alpha = animatedProfileAlpha)
+                ) {
+                    // for each item in the list will be calculated the position of the item
+                    // lazyGrid is not used because it is not possible to calculate correctly the position of the item
+                    onSetScreenSize(
+                        this.maxWidth.value,
+                        this.maxHeight.value,
+                        this.maxWidth.dpToPx(),
+                        this.maxHeight.dpToPx()
+                    )
+
+                    profiles.forEach { profile ->
+                        val profileItemPosition = if (offsetProfiles.isNotEmpty()) {
+                            with(offsetProfiles[profiles.indexOf(profile)]) {
+                                IntOffset(first, second)
+                            }
+                        } else {
+                            IntOffset(0, 0)
+                        }
+
+                        Column(
+                            horizontalAlignment = Alignment.CenterHorizontally,
+                            modifier = Modifier
+                                .offset {
+                                    profileItemPosition
+                                }
+                        ) {
+                            AsyncImage(
+                                model = profile.imageUrl,
+                                placeholder = painterResource(id = R.drawable.image_placeholder),
+                                contentDescription = stringResource(
+                                    id = R.string.profile_current_profile_name,
+                                    profile.name
+                                ),
+                                modifier = Modifier
+                                    .clip(RoundedCornerShape(5))
+                                    .alpha(if (selectedItem == profile) selectedImageAlpha else 1f)
+                                    .size(defaultImageSize.dp)
+                                    .clickable {
+                                        onClickSelectedProfile(profile)
+                                    }
+                            )
+
+                            Text(
+                                text = profile.name,
+                                style = MaterialTheme.typography.bodyMedium.copy(
+                                    color = MaterialTheme.colorScheme.onSurface
+                                ),
+                                modifier = Modifier.padding(top = 8.dp)
+                            )
+
+                            Spacer(modifier = Modifier.height(28.dp))
+                        }
+                        if (profiles.indexOf(profile) == profiles.size - 1) {
```

Comment:
quando aconteceria isso?

Reply:
Como eu estou posicionando cada item perfil manualmente com o `profiles.forEach`, ao invés de usar um lazygrid, quando o ultimo for adicionado a posição correta, esse if informa o viewModel para mostrar todos de uma vez.